XCommas.Net/XCommas.Net/XCommasApi.cs

```diff
@@ -453,6 +453,20 @@ public async Task<XCommasResponse<SmartTradeStep>> AddFundsToSmartTradeAsync(Sma
         }
         #endregion
 
+        #region users
+
+        public XCommasResponse<bool> ChangeUserMode(User mode) => this.ChangeUserModeAsync(mode).Result;
+        public async Task<XCommasResponse<bool>> ChangeUserModeAsync(User mode)
+        {
+            var path = $"{BaseAddress}/ver1/users/change_mode?mode=" + mode;
```

Comment:
Proper string interpolation should be used, eg:
var path = $"{BaseAddress}/ver1/users/change_mode?mode={mode}";